cypress/support/commands.js

```diff
@@ -23,3 +24,88 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+/** Initialize an empty document w/ ReactDOM and DOM events.
+    @function   cy.injectReactDOM
+**/
+Cypress.Commands.add('injectReactDOM', () => {
+  var packages = {}
+  // Yeah, due to how Cy resolves promises under the hood, a closure works but an @aliased cached asset doesn't
+  return cy
+    .log('Injecting ReactDOM for Unit Testing')
+    .readFile('node_modules/react/umd/react.development.js', { log: false }).then(file => packages.React = file)
```

Comment:
for performance reasons you might read these two files once and just keep them in memory